### PR TITLE
Flux v0.125.0

### PR DESCRIPTION
**Pull request by sanderson**
- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/webexteams/endpoint.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+_For more information, see [`webexteams.message()` parameters](/influxdb/v2.0/reference/flux/stdlib/contrib/webexteams/message/#parameters)._
+
+## Examples
+
+##### Send the last reported status to Webex Teams
+```js
+import "contrib/sranka/webexteams"
+import "influxdata/influxdb/secrets"
+
+// this value can be stored in the secret-store()
```
Comment (jstirnaman) [inline content/influxdb/v2.0/reference/flux/stdlib/contrib/webexteams/endpoint.md:65]: I prefer JSDOC (block) comment syntax. I know it's more verbose, but it's a visual indicator that I should read it, not ignore it as commented code.

```suggestion
/**
 * Use the secret-store() to store third-party tokens.
 */
```

File: content/influxdb/v2.0/reference/release-notes/flux.md [hunk 2]
```diff
@@ -8,6 +8,23 @@ menu:
...
+- Add feature flag library as an internal package.
+- Add narrow transformation transport.
+- Add transport-aware dataset.
+- Simplify the transport interface and add a transformation adapter.
+- Add [`contrib/sranka/webexteams` package](/influxdb/v2.0/reference/flux/stdlib/contrib/webexteams/).
+- Add optimized repeat function for arrow arrays.
+- Add two additional internal message types.
+
+### Bug fixes
+- Update transformation adapter to correctly error when receiving a flush key
```
Comment (jstirnaman) [inline content/influxdb/v2.0/reference/release-notes/flux.md:23]:
```suggestion
- Update transformation adapter to return an error when receiving a flush key
```

**Review by jstirnaman — COMMENTED** [review]
Minor suggestions, otherwise looks good.

File: content/influxdb/v2.0/reference/flux/stdlib/contrib/webexteams/endpoint.md (hunk 1 above)
Comment (sanderson) [reply content/influxdb/v2.0/reference/flux/stdlib/contrib/webexteams/endpoint.md:65]: Yeah, Flux doesn't support block comments: https://docs.influxdata.com/influxdb/v2.0/reference/flux/language/lexical-elements/#comments.

I meant to remove this line altogether anyway. 
```suggestion
```

**Pull request merged**